src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -12,8 +12,8 @@
         novalidate>
     <div class="panel panel-default">
       <div class="panel-heading">
-        <h3 class="panel-title">
-          <span i18n>{editing, select, 1 {Edit} other {Add}} pool</span>
+        <h3 class="panel-title" i18n="form title|Example: Create Pool@@formTitle">
```

Comment:
I would suggest you remove the `@@formTitle` or use the same meaning for both `formTitle` and `formActionButton`. 
Currently we get 2 different `trans-units`, just because the meaning is different.

Reply:
Not so sure about this. Why? I'm using `{{ action | titlecase }} {{ resource | titlecase }}` for both a descriptive element (Form title: "Create Pool") and for imperative one (Button label: "Create Pool"), and I chose the `titlecase` capitalization because I was following this [recommendation](https://www.patternfly.org/styles/terminology-and-wording/#capitalization), which only takes English grammar into consideration.

Other languages might follow different practices (e.g.: nouns in German are always in uppercase, while in Spanish headline capitalization is incorrect, and I cannot imagine what non-indoeuropean languages would do about this). I prefer to keep room for a translator to deal with each case separately.

Reply:
Transifex doesn't allow you to change the interpolations.
The translator will only see `<1> <2>` and can only change the order of the 2 words, they will not be able to change the capitalization.
p.e.: https://www.transifex.com/ceph/ceph-dashboard/translate/#pt_PT/messagesxlf/164387632

Reply:
Oops, true! I also checked in [the ICU](http://userguide.icu-project.org/transforms/casemappings) for the case the had some way to deal with that, but I don't see anything specific to Titlecase/Headings.

However, I still prefer to keep 2 trans-units for this (even if they are _right now_ exactly the same), as they semantically 2 different things. With this change we have already reduced 2 x 2 x 8 (32) translation units to just 2.